src/components/Footer/Footer.jsx

```diff
@@ -0,0 +1,50 @@
+import "./Footer.css";
+import { Link } from "react-router-dom";
+import Facebook from "../../assets/facebook.svg";
+import Github from "../../assets/github.svg";
+function Footer() {
+  return (
+    <footer className="footer">
+      <div className="footer__container">
+        <p className="footer__copyright">
+          © 2024 Supersite, Powered by News API
+        </p>
+        <div className="footer__links">
+          <div className="footer__links-container">
+            <Link to="/" className="footer__link--home">
+              Home
+            </Link>
+            <a
+              href="https://tripleten.com/"
+              target="_blank"
+              rel="noopener noreferrer"
+              className="footer__link--tripleten"
+            >
+              TripleTen
+            </a>
+          </div>
+          <div className="footer__social">
+            <a
+              href="https://github.com/mouachee"
+              className="footer__social--github"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              <img src={Github} alt="" className="footer__social-icon" />
```

Comment:
`alt=""` 

Every `img` tag must have an **descriptive** `alt` attribute. The `alt` attribute provides **alternative information** for an image if a user, for some reason, cannot view it (because of slow connection, an error in the `src` attribute, or if the user is using a screen reader). Well-written `alt` text is important to your website's accessibility, and its search engine optimization. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img https://moz.com/learn/seo/alt-text

**Please, correct it everywhere in the code**